rule/src/main/java/com/buschmais/jqassistant/core/rule/impl/reader/IndentHelper.java

```diff
@@ -0,0 +1,124 @@
+package com.buschmais.jqassistant.core.rule.impl.reader;
+
+public class IndentHelper {
+    /**
+     * This class make the String more effective and understandable. by removing the extra spaces,lines and tabs.
```

Comment:
Describe the acutal behavior of the method, i.e. that it removes leading and trailing blank lines and removes indent of lines containing text.